cli/main.rs

```diff
@@ -854,16 +854,25 @@ async fn test_command(
   let mut worker =
     MainWorker::new(&program_state, main_module.clone(), permissions);
 
-  let mut maybe_coverage_collector = if flags.coverage {
-    let session = worker.create_inspector_session();
-    let mut coverage_collector =
-      tools::coverage::CoverageCollector::new(session);
-    coverage_collector.start_collecting().await?;
+  // If the actual coverage flag is set, we set the environment variable so that subprocesses will
+  // forward it.
+  if let Some(ref coverage_dir) = flags.coverage_dir {
+    env::set_var("DENO_COVERAGE_DIR", coverage_dir);
```

Comment:
Rename to something obnoxious, like `DENO_UNSTABLE_INTERNAL_COVERAGE_DIR`?

Reply:
Yes

Reply:
Also is this env var unset after coverage is done?